src/studio-header/StudioHeader.jsx

```diff
@@ -53,6 +54,7 @@ StudioHeader.propTypes = {
   number: PropTypes.string,
   org: PropTypes.string,
   title: PropTypes.string.isRequired,
+  containerProps: HeaderBody.propTypes.containerProps,
```

Comment:
`containerProps` should be optional, and/or populated with a default value below?

Reply:
Fixed here: 41d76ccda54c89b144f13896e8835d2141d4483d